react/src/components/base/Typography/Typography.stories.mdx

```diff
@@ -19,120 +19,143 @@ const slugifyStyle = (style) => {
-          <hr />
-        </div>
-        <div className="sb-block">
-          Source Code Pro is an open source font available on Google Fonts.
-          <DecorativeLink
-            text="Download the fonts or see more information on Google Fonts"
-            href="https://fonts.google.com/specimen/Source+Code+Pro"
-          />
-        </div>
-    </div>
-  </Story>
-</Canvas>
+## Texta
+
+**The official typeface of Mass.gov is Texta. We picked Texta for the Mass.gov branding purposes.**
+
+![&quot;Mass.gov&quot; displaying in Texta](https://raw.githubusercontent.com/massgov/mayflower/develop/docs/.gitbook/assets/texta-massgov.png)
+
+![&quot;MA&quot; displaying in Texta](https://raw.githubusercontent.com/massgov/mayflower/develop/docs/.gitbook/assets/texta-ma.png)
+
+**Texta is a versatile, readable font that supports 23 different languages.** They are: Albanian, Alsatian, Aragonese, Arapaho, Arrernte, Asturian, Aymara, Basque, Bislama, Breton, Cebuano, Chamorro, Cheyenne, Corsican, Danish, Dutch, English, Estonian, Faroese, Fijian, Finnish, French, French Creole \(Saint Lucia\), Frisian, Galician, Genoese, German, Gilbertese \(Kiribati\), Greenlandic, Haitian Creole, Hiligaynon, Hmong, Hopi, Ibanag, Iloko \(Ilokano\), Indonesian, Interglossa \(Glosa\), Interlingua, Irish \(Gaelic\), Islandic, Italian, Jèrriais, Lojban, Lombard, Luxembourgian, Malagasy, Manx, Mohawk, Norfolk/Pitcairnese, Northern Sotho \(Pedi\), Norwegian, Occitan, Oromo, Pangasinan, Papiamento, Piedmontese, Portuguese, Potawatomi, Rhaeto-Romance, Romansh \(Rumantsch\), Rotokas, Sami \(Lule\), Samoan, Sardinian \(Sardu\), Scots \(Gaelic\), Seychellois Creole \(Seselwa\), Shona, Sicilian, Somali, Southern Ndebele, Southern Sotho \(Sesotho\), Spanish, Swahili, Swati/Swazi, Swedish, Tagalog \(Filipino/Pilipino\), Tetum \(Tetun\), Tok Pisin, Tswana, Uyghur \(Latinized\), Volapük, Votic \(Latinized\), Walloon, Warlpiri, Xhosa, Yapese and Zulu.
+
+### Usage
+
+Texta font has the following font weight options:
+
+| Font weight | Specs |
+| :--- | :--- |
+| Light | 300 |
+| Medium | 500 |
+| Bold | 700 |
+| Black | 800 |
+
+All font files can be downloaded either from the [**Mayflower Github repo**](https://github.com/massgov/mayflower/tree/develop/assets/fonts) or from the [**Mayflower NPM package**](https://unpkg.com/@massds/mayflower@6.0.0/fonts/). Please read the web font [**web font license**](https://www.fontspring.com/lic/htswufoczd) **\*\*carefully to proceed**.\*\*
+
+### License
+
+Fontspring grants Licensee a perpetual, worldwide, non-exclusive and non-transferrable license to install the Texta as webfont **on Websites that are owned by the Commonwealth of Massachusetts** using the @font-face selector in CSS files. For other usage, you may have to acquire an additional license through Fontsprint.
+
+For more details about Texta's terms of use, please refer to our [**web font license**](https://www.fontspring.com/lic/htswufoczd).
+
+
+
+<div className="page-content">
```

Comment:
Can we break source code pro into a different story. Otherwise I like combining the Texta stories